tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -314,35 +314,21 @@ def test_end_to_end(self):
                         common_locators["alert.success"]))
             # Create VM
             with VirtualMachine(distro='rhel66') as vm:
-                # Download and Install rpm
-                result = vm.run(
-                    "wget -nd -r -l1 --no-parent -A '*.noarch.rpm' "
-                    "http://{0}/pub/".format(self.server_name)
-                )
-                self.assertEqual(
-                    result.return_code, 0,
-                    "failed to fetch katello-ca rpm: {0}, return code: {1}"
-                    .format(result.stderr, result.return_code)
-                )
-                result = vm.run(
-                    'rpm -i katello-ca-consumer*.noarch.rpm'
-                )
-                self.assertEqual(
-                    result.return_code, 0,
-                    "failed to install katello-ca rpm: {0}, return code: {1}"
-                    .format(result.stderr, result.return_code)
-                )
-                # Register client with foreman server using activation-key
-                result = vm.run(
-                    'subscription-manager register --activationkey {0} '
-                    '--org {1} --force'
-                    .format(activation_key_name, org_name)
-                )
+                # Install katello cert
+                vm.install_katello_cert()
+
+                # Register client with satellite server using activation-key
+                result = vm.register_contenthost(
+                    activation_key=activation_key_name,
+                    org=org_name)
```

Comment:
I followed the convention specified in https://www.python.org/dev/peps/pep-0008/#indentation